docs/source/user/api_change.rst

```diff
@@ -46,6 +58,7 @@ OpenFAST                                      28   Patm            103500     Pa
 OpenFAST                                      29   Pvap            1700       Pvap              - Vapour pressure of working fluid (Pa) [used only for an MHK turbine cavitation check]
 OpenFAST                                      30   WtrDpth         50         WtrDpth           - Water depth (m)
 OpenFAST                                      31   MSL2SWL         0          MSL2SWL           - Offset between still-water level and mean sea level (m) [positive upward]
+TurbSim                                       13   WrHAWCFF        False      WrHAWCFF          - Output full-field time-series data in HAWC form?  (Generates RootName-u.bin, RootName-v.bin, RootName-w.bin, RootName.hawc)
```

Comment:
WrHAWCFF is a new input added in OpenFAST v3.2.0, and should not show up in the v3.1.0 table.

Reply:
See #1178 for correction